package.json

```diff
@@ -48,7 +48,7 @@
   "homepage": "https://github.com/DanielRuf/website-checks#readme",
   "dependencies": {
     "kleur": "3.0.2",
-    "puppeteer": "1.13.0"
+    "puppeteer": "^1.19.0"
```

Comment:
We have updated the master branch. Can you rebase on it? If not I can do that.

Reply:
I will give it a shot.

Reply:
Is it fine now?

Reply:
Perfect =)